src/librustdoc/html/render/write_shared.rs

```diff
@@ -464,8 +464,6 @@ pub(super) fn write_shared(
     // Update the list of all implementors for traits
     let dst = cx.dst.join("implementors");
     for (&did, imps) in &cx.cache.implementors {
-        let did = did.expect_real();
```

Comment:
Why removing this?

Reply:
Ah nevermind, we changed `implementors` into `DefId`, so there is no need for that.